src/aks-preview/azext_aks_preview/tests/latest/recordings/test_aks_byo_appgw_with_ingress_appgw_addon.yaml

```diff
@@ -1790,44 +1825,62 @@ interactions:
+      Content-Length:
+      - '65'
+      Content-Type:
+      - application/json; charset=utf-8
       ParameterSetName:
-      - -n -g --enable-managed-identity --vnet-subnet-id -a --appgw-id -o
+      - -n -g --enable-managed-identity --service-principal --client-secret --generate-ssh-keys
+        --vnet-subnet-id -a --appgw-id -o
       User-Agent:
-      - python/3.8.0 (Linux-5.3.0-1020-azure-x86_64-with-glibc2.2.5) msrest/0.6.11
-        msrest_azure/0.6.2 azure-mgmt-containerservice/4.4.2 Azure-SDK-For-Python
-        AZURECLI/2.6.0
-    method: GET
-    uri: https://management.azure.com/subscriptions/00000000-0000-0000-0000-000000000000/providers/Microsoft.ContainerService/locations/westus2/operations/8abc5190-89c5-474b-b6dc-362530799bcf?api-version=2016-03-30
+      - python/3.6.9 (Linux-5.0.0-1032-azure-x86_64-with-Ubuntu-18.04-bionic) msrest/0.6.9
+        msrest_azure/0.6.3 azure-graphrbac/0.60.0 Azure-SDK-For-Python AZURECLI/2.8.0
+      accept-language:
+      - en-US
+    method: POST
+    uri: https://graph.windows.net/00000000-0000-0000-0000-000000000000/getObjectsByObjectIds?api-version=1.6
   response:
     body:
-      string: "{\n  \"name\": \"9051bc8a-c589-4b47-b6dc-362530799bcf\",\n  \"status\":
-        \"InProgress\",\n  \"startTime\": \"2020-05-26T17:14:30.8101275Z\"\n }"
+      string: '{"odata.error":{"code":"Request_BadRequest","message":{"lang":"en","value":"Invalid
+        GUID specified."},"requestId":"78ce592f-3996-43f7-9255-dcc6fe4b2191","date":"2020-07-02T08:56:48"}}'
     headers:
+      access-control-allow-origin:
+      - '*'
       cache-control:
       - no-cache
       content-length:
-      - '126'
+      - '183'
       content-type:
-      - application/json
+      - application/json; odata=minimalmetadata; streaming=true; charset=utf-8
+      dataserviceversion:
+      - 3.0;
       date:
-      - Tue, 26 May 2020 17:15:31 GMT
+      - Thu, 02 Jul 2020 08:56:47 GMT
+      duration:
+      - '519688'
       expires:
       - '-1'
+      ocp-aad-diagnostics-server-name:
+      - tXMxN0F2O1FjGGhO67DGJT21hizmwNrxQUHId/gayo4=
+      ocp-aad-session-key:
+      - W3zvm7ARxjUZw4KZUYhKQd-TshBh1MMhJ_N_1_yQ4d9F-IsPVFE8fMVbbdl42eGkFOzfRHXFllWcB7w9q5J2IcDptQXGJH1I0tMarxpoesZTAo9BV2faZ6y-JiYF5pAdBGput2z-IoIikByAm_6FMibqWWgz94PYYR2oqewMG4XKH4Bpkus-u5tyufS0ISHHTMjRNSdnEUFS5SFwREZ35A.AdOpRn8xsgIZLzZlbQPncZiyFih-Ih6KEZ9I96EGN4c
       pragma:
       - no-cache
-      server:
-      - nginx
+      request-id:
+      - 78ce592f-3996-43f7-9255-dcc6fe4b2191
       strict-transport-security:
       - max-age=31536000; includeSubDomains
-      transfer-encoding:
-      - chunked
-      vary:
-      - Accept-Encoding
-      x-content-type-options:
-      - nosniff
+      x-aad-resource-unit:
+      - '3'
+      x-aspnet-version:
+      - 4.0.30319
+      x-ms-dirapi-data-contract-version:
+      - '1.6'
+      x-powered-by:
+      - ASP.NET
     status:
-      code: 200
-      message: OK
+      code: 400
```

Comment:
Is "bad request" expected?

Reply:
not sure, i ran the tests again just to be sure, but getting the same